src/lib/gundb/UserStorageClass.js

```diff
@@ -1147,7 +1152,7 @@ export class UserStorage {
           .get('avatar')
           .get('display')
           .then())) ||
-      (type === 'claim' || address === '0x0000000000000000000000000000000000000000'
+      (['claim', 'bonus'].includes(type) || address === '0x0000000000000000000000000000000000000000'
```

Comment:
please, use constant

Reply:
@StanislavShevchenko see new commit with the changes.